tests/integration/common.sh

```diff
@@ -220,15 +220,33 @@ form_network() {
     set -e
     local pskc=$1
 
+    echo "------ forming Thread network ------"
+
+    echo "sudo ${OT_CTL} channel ${CHANNEL}"
```

Comment:
Being unfamiliar with the reason of adding these logging, should we use `set -x` instead ?

Reply:
changed to use `set -x`.